### PR TITLE
feature/article-link-by-default-BZ-6411

### PR DESCRIPTION
**Pull request by tiandavis**
## Summary - [BZ-6411](https://thirdiron.atlassian.net/browse/BZ-6411)

Show article link by default.

## Deploy Precautions

- None

### CODE REVIEW

File: src/summon/browzine-summon-adapter.js [hunk 1]
```diff
@@ -383,7 +383,7 @@ browzine.summon = (function() {
     var featureEnabled = false;
     var config = browzine.articleLinkEnabled;
 
-    if (config === true) {
+    if (typeof config === "undefined" || config === null || config === true) {
```
Comment (karlbecker) [inline src/summon/browzine-summon-adapter.js:386]: Works for me 👍 

We could also just change it to check if `config === false` perhaps? But no biggie, this definitely gets the job done, and should be sure not to break the existing installs.

**Review by karlbecker — APPROVED** [review]
Simple change, LGTM @tiandavis !

File: src/summon/browzine-summon-adapter.js (hunk 1 above)
Comment (tiandavis) [reply src/summon/browzine-summon-adapter.js:386]: > We could also just change it to check if config === false perhaps?

One would think, but alas JavaScript...
image

The current setup helps us account for when the `articleLinkEnabled` property doesn't exist at all. e.g `undefined` mainly, but possibly even a `null` value.

Comment (karlbecker) [reply src/summon/browzine-summon-adapter.js:386]: JavaScript!
iu-16

Thanks for ensuring we didn't ship my wrong idea 😄

**Pull request merged**